lib/common/output.c

```diff
@@ -216,8 +216,8 @@ pcmk__xml_output_new(pcmk__output_t **out, xmlNodePtr *xml) {
  * \internal
  * \brief  Finish and free an XML-only output object
  *
- * \param[in]  out     Output object to free
- * \param[out] xml     Where to store XML output
+ * \param[in,out] out  Output object to free
+ * \param[out]    xml  Where to store XML output
```

Comment:
I think we should something like `if not NULL` for `xml`

Reply:
good point